src/ts-transformer/css-prop/__tests__/index.test.tsx

```diff
@@ -156,27 +159,25 @@ describe('css prop transformer', () => {
       expect(actual).toInclude('<style>.test-class:hover{color:blue;}</style>');
     });
 
-    it('should transform object with object selector from variable', async () => {
-      const actual = await fullTransform({
-        index: `
-          /** @jsx jsx */
-          import { jsx } from '${pkg.name}';
-          import { mixin } from './mixins';
-
-          <div
-            css={{
-              display: 'flex',
-              fontSize: '50px',
-              color: 'blue',
-              ':hover': mixin,
-            }}>
-            Hello, world!
-          </div>
-        `,
-        mixins: `
-          export const mixin = { color: 'blue' };
-        `,
-      });
+    it('should transform object with object selector from variable', () => {
+      const actual = transformer.addSource({
```

Comment:
when you add source does it mutate the local above (does every test now have a source?)

Reply:
Currently it does, I'll change the implementation to create new instances for each call to maintain proper encapsulation here.